binding/webgl_rendering_context.cc

```diff
@@ -2466,6 +2576,82 @@ napi_value WebGLRenderingContext::Uniform2i(napi_env env,
   return nullptr;
 }
 
+/* static */
+napi_value WebGLRenderingContext::Uniform2iv(napi_env env,
+                                             napi_callback_info info) {
+  LOG_CALL("Uniform2iv");
+  napi_status nstatus;
+
+  size_t argc = 2;
+  napi_value args[2];
+  napi_value js_this;
+  nstatus = napi_get_cb_info(env, info, &argc, args, &js_this, nullptr);
+  ENSURE_NAPI_OK_RETVAL(env, nstatus, nullptr);
+  ENSURE_ARGC_RETVAL(env, argc, 2, nullptr);
+
+  ENSURE_VALUE_IS_NUMBER_RETVAL(env, args[0], nullptr);
+
+  GLint location;
+  nstatus = napi_get_value_int32(env, args[0], &location);
+  ENSURE_NAPI_OK_RETVAL(env, nstatus, nullptr);
+
+  size_t size;
+  void *data;
+  nstatus = napi_get_typedarray_info(env, args[1], nullptr, &size, &data,
```

Comment:
Ditto here about type checking. A user can pass in a regular array as well as a typed array for values.

Reply:
Added `TODO`

Reply:
Can you add a check for object/typedarray like you did in `::BufferSubData()`?